### PR TITLE
[openspecfun] Update to 0.5.4

### PR DESCRIPTION
**Pull request by ViralBShah**

### CODE REVIEW

**Comment by ViralBShah** [comment]
I renamed the master branch to main - is that why it is not liking it?

File: O/OpenSpecFun/build_tarballs.jl [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 name = "OpenSpecFun"
-version = v"0.5.3"
+version = v"0.5.4"
 
 # Collection of sources required to build openspecfun
 sources = [
-    ArchiveSource("https://github.com/JuliaMath/openspecfun/archive/v0.5.3.tar.gz",
-                  "1505c7a45f9f39ffe18be36f7a985cb427873948281dbcd376a11c2cd15e41e7"),
+    GitSource("https://github.com/JuliaMath/openspecfun/",
```
Comment (giordano) [inline O/OpenSpecFun/build_tarballs.jl:8]:
```suggestion
    GitSource("https://github.com/JuliaMath/openspecfun.git",
```

**Comment by giordano** [comment]
I think LibGit2 didn't like the format of the repository URL

**Comment by ViralBShah** [comment]
Merge?

**Pull request merged**